examples/templates/devcontainer-docker/main.tf

```diff
@@ -145,9 +149,18 @@ resource "docker_volume" "workspaces" {
   }
 }
 
+# Check for the presence of a prebuilt image in the cache repo
+# that we can use instead.
+resource "envbuilder_cached_image" "cached" {
+  count         = data.coder_workspace.me.start_count
```

Comment:
Is the count needed here, do we need it to be destroyed to invalidate any caching?

Reply:
I essentially don't want to slow down stopping a workspace.
When you read the resource it will check for the existence of the previously set image in the state. If that can't be found, then the plan will force it to be replaced.